src/ReleaseHistory.md

```diff
@@ -1,5 +1,12 @@
 # Microsoft Json Schema Packages
 
+## **1.1.4** [Pointer](https://www.nuget.org/packages/Microsoft.Json.Pointer/1.1.4) | [Schema](https://www.nuget.org/packages/Microsoft.Json.Schema/1.1.4)| [Schema.ToDotNet](https://www.nuget.org/packages/Microsoft.Json.Schema.ToDotNet/1.1.4)| [Schema.Validation](https://www.nuget.org/packages/Microsoft.Json.Schema.Validation/1.1.4)
```

Comment:
>**1.1.4** [](http://example.com/codeflow?start=3&length=9)

change to unreleased, since we dont know yet when we will ship this version

Reply:
fixed